.circleci/config.yml

```diff
@@ -92,6 +92,8 @@ jobs:
       - run:
           name: "Preparing environment"
           command: |
+            sudo apt-add-repository ppa:fish-shell/release-3
```

Comment:
Please add a comment here explaining why we need this.

Reply:
I think I parsed their git log incorrectly. The space fix is in master but not the latest release yet. We can test against a nightly build now and use the official release once a new version is out.

Reply:
ah, it did seem a bit too fantastic to me :)
can we automatically install nightly?

Reply:
if not, we can skip add a skip for now and re-enable it once they have a proper release.

Reply:
Indeed managing their releases is a bit overkill. I got the code to run tests only with fish > 3.1.2 or git build after 3.1.2. The latter is for local tests.